index.js

```diff
@@ -93,7 +99,11 @@
       log.debug('Processing "%s".', file.originalPath);
       if(shouldStop(report)) {
         done(report.results);
+      } else if(options.stopAboveErrorThreshold && totalErrorCount > options.errorThreshold) {
+        log.error('\n' + chalk.red('There are more than ' + options.errorThreshold + ' errors'));
+        process.exit();
       } else {
+        log.error('\n' + chalk.red('There are ' + totalErrorCount + ' errors'));
```

Comment:
Will this be logged even if there are `0` errors?

If so, you might want to not show it if error count is `0` or display a green message rather than red which suggests an error.